### PR TITLE
(profile::pi::rubinhat) fwv 

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

**Review by wvreeven — APPROVED** [review]
Looks good to me.

File: spec/support/spec/rubinhab.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
```
Comment (fritzm) [inline spec/support/spec/rubinhab.rb]: Typo on file name?

**Review by fritzm — APPROVED** [review]
One typo'd filename?  Otherwise LGTM.

File: spec/support/spec/rubinhab.rb (hunk 1 above)
Comment (jhoblitt) [reply spec/support/spec/rubinhab.rb]: Fixed.  The name of the file actually isn't significant as every file under `spec/support/spec` is loaded.

**Pull request merged**